composer.lock

```diff
@@ -8,7 +8,7 @@
     "packages": [
         {
             "name": "automattic/jetpack-asset-tools",
-            "version": "dev-master",
+            "version": "dev-add/clean-composer-script",
```

Comment:
Could we revert all the `composer.lock` changes? They seem to be unnecessary and occurred only because we ran `composer update` while in the branch.

Reply:
isn't this preferred? To commit composer.lock updates along with the changes themselves? To avoid PR's that only update composer.lock?

Reply:
Ok, manually updated this to stay pointing to dev-master